src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -22,576 +22,574 @@ internal class StyleViewModel : AbstractOptionPreviewViewModel
-    public int Id { get; set; }
+        private static readonly string s_propertyDeclarationPreviewFalse = $@"
+class C{{
+    public int {ServicesVSResources.Id} {{ get; set; }}
     void Method()
-    {
+    {{
 //[
-        Id = 0;
+        {ServicesVSResources.Id} = 0;
 //]
-    }
-}";
+    }}
+}}";
 
-        private static readonly string s_eventDeclarationPreviewTrue = @"
+        private static readonly string s_eventDeclarationPreviewTrue = $@"
 using System;
-class C{
-    event EventHandler Elapsed;
-    void Handler(object sender, EventArgs args)
-    {
+class C{{
+    event EventHandler {ServicesVSResources.Elapsed};
+    void {ServicesVSResources.Handler}(object sender, EventArgs args)
+    {{
 //[
-        this.Elapsed += Handler;
+        this.{ServicesVSResources.Elapsed} += {ServicesVSResources.Handler};
 //]
-    }
-}";
+    }}
+}}";
 
-        private static readonly string s_eventDeclarationPreviewFalse = @"
+        private static readonly string s_eventDeclarationPreviewFalse = $@"
 using System;
-class C{
-    event EventHandler Elapsed;
-    void Handler(object sender, EventArgs args)
-    {
+class C{{
+    event EventHandler {ServicesVSResources.Elapsed};
+    void {ServicesVSResources.Handler}(object sender, EventArgs args)
+    {{
 //[
-        Elapsed += Handler;
+        {ServicesVSResources.Elapsed} += {ServicesVSResources.Handler};
 //]
-    }
-}";
+    }}
+}}";
 
-        private static readonly string s_methodDeclarationPreviewTrue = @"
+        private static readonly string s_methodDeclarationPreviewTrue = $@"
 using System;
-class C{
-    void Display()
-    {
+class C{{
+    void {ServicesVSResources.Display}()
+    {{
 //[
-        this.Display();
+        this.{ServicesVSResources.Display}();
 //]
-    }
-}";
+    }}
+}}";
 
-        private static readonly string s_methodDeclarationPreviewFalse = @"
+        private static readonly string s_methodDeclarationPreviewFalse = $@"
 using System;
-class C{
-    void Display()
-    {
+class C{{
+    void {ServicesVSResources.Display}()
+    {{
 //[
-        Display();
+        {ServicesVSResources.Display}();
 //]
-    }
-}";
+    }}
+}}";
 
-        private static readonly string s_intrinsicPreviewDeclarationTrue = @"
+        private static readonly string s_intrinsicPreviewDeclarationTrue = $@"
 class Program
-{
+{{
 //[
-    private int _member;
-    static void M(int argument)
-    {
-        int local;
-    }
+    private int {ServicesVSResources.Underscore_member};
+    static void M(int {ServicesVSResources.argument})
+    {{
+        int {ServicesVSResources.local};
+    }}
 //]
-}";
+}}";
 
-        private static readonly string s_intrinsicPreviewDeclarationFalse = @"
+        private static readonly string s_intrinsicPreviewDeclarationFalse = $@"
 using System;
 class Program
-{
+{{
 //[
-    private Int32 _member;
-    static void M(Int32 argument)
-    {
-        Int32 local;
-    }
+    private Int32 {ServicesVSResources.Underscore_member};
```

Comment:
Along those lines, I think that localizing the comments alone might have been sufficient. 

Regardless, I don't think we should use one-letter identifiers just to side-step the issue.